README.md

```diff
@@ -2,4 +2,4 @@
 
 > Native embedded bindings to the libsodium cryptography library 
 
-This library vendors libsodium so that you don't have to.
+This library vendors libsodium so that you don't have to. 
```

Comment:
Extraneous space?

Reply:
I put that in to re-run the actions to check how quickly our linting and formatting runs 'hot' versus 'cold'. Can remove.

Reply:
ah great :)